src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { searchArticles } from "../../utils/NewsApi";
+
+export default function SearchForm({
+    setArticles,
+    setIsLoading,
+    setSearchQuestion
+}) {
+    const [value, setValue] = useState({ question: "" })
```

Comment:
**Might be better**

You can make a custom hook for inputs handling:

```js
export function useForm(inputValues) {
  const [values, setValues] = useState(inputValues);

  const handleChange = (event) => {
    const {value, name} = event.target;
    setValues({...values, [name]: value});
  };
  return {values, handleChange, setValues};
}

```

This way you can easily handle any form in your project

And now only one line of code enables form handling:

```jsx

const {values, handleChange, setValues} = useForm({})


```